oak-run/pom.xml

```diff
@@ -336,10 +336,6 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
```

Comment:
Why are you removing these dependencies?

Reply:
restored